includes/RequestWiki/SpecialRequestWiki.php

```diff
@@ -95,6 +103,15 @@ protected function getFormFields() {
 			];
 		}
 
+		if ( $this->config->get( 'CreateWikiUseWikiTags' ) ) {
+			$formDescriptor['wikitags'] = [
+				'type' => 'text',
+				'label-message' => 'createwiki-label-wiki-tags',
+				'cssclass' => 'createwiki-wikitags',
+				'disabled' => true,
```

Comment:
This should be `readonly => true`, no? Or else it won't be sent with the form?

Reply:
You may be right. This still has a lot of work like also passing to cw_requests also